spec/services/prison_visits/api_spec.rb

```diff
@@ -135,10 +135,6 @@
   describe 'request_visit', vcr: { cassette_name: 'request_visit' } do
     subject { super().request_visit(valid_booking_params) }
 
-    before do
```

Comment:
Without this test we don't really check that uses different locales because `I18n.locale` defaults to `en`